tests/pillar/horizon_with_ssl_extra2.sls

```diff
@@ -0,0 +1,39 @@
+salt:
+  minion:
+    enabled: true
+nginx:
+  server:
+    enabled: true
+    extras: false
+    bind:
+      address: 127.0.0.1
+      protocol: tcp
+    site:
+      # TODO: if site01 enabled in this config then ci fails
```

Comment:
This is something that bit worries me. But no idea why it actually happen. Topic for further PR.